offer-manager-domain/src/main/groovy/life/qbic/business/products/create/CreateProduct.groovy

```diff
@@ -0,0 +1,12 @@
+package life.qbic.business.products.create
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
+ *
+ */
```

Comment:
```suggestion
/**
 * <h1>4.3.0 Create Service Product</h1>
 * <br>
 * <p> When the service portfolio changed due to a business decision, as an offer administrator, I want to provide information on the new service offered and make it available to new offers upon creation. 
 * </p>
 *
 * @since: 1.0.0
 */
```